acj/__init__.py

```diff
@@ -84,4 +92,29 @@ def bouncer_user_loader():
 
     return app
 
+def create_worker_app(conf=config, settings_override=None):
+    """Return a :class:`Flask` application instance
+
+    :param conf: Flask config object
+    :param settings_override: override the default settings or settings in the configuration file
+    """
+    if settings_override is None:
+        settings_override = {}
+    app = Flask(__name__, static_url_path='/app')
+    app.config.update(conf)
+    app.config.update(settings_override)
+
+    app.logger.debug("Application Configuration: " + str(app.config))
+
+    db.init_app(app)
+
+    celery = Celery(
+        app.name,
+        backend=app.config['CELERY_RESULT_BACKEND'],
+        broker=app.config['CELERY_BROKER_URL']
+    )
+    celery.conf.update(app.config)
```

Comment:
Looks like line 101 to 116  is a duplicate from create_app function. Is it possible to extract into a separate function and call it in both create functions.